### ISSUE TITLE
new bin options

### ISSUE THREAD

**Issue by SharkBairRFSR** [mekanism/Mekanism-Feature-Requests#305]
### Describe the feature you'd like

I would like to see two new bins added to mekanism. The first would be a void upgrade for a bin like in storage drawers, perhaps add a bucket of lava to the bin recipe and it now has a void upgrade holding the normal amount and voiding excess. the second would be a trash bin that can destroy its contents.

**Comment by pupnewfster** [comment]
I will leave this open for now but see #199 for my thoughts on both of those suggestions.

**Comment by zontreck** [comment]
Perhaps one solution would be to make the "Creative" empty containers be able to be crafted. Those are already void containers. Maybe using antimatter or something to explain away the disappearing matter inside it?

**Comment by pupnewfster** [comment]
No. https://github.com/mekanism/Mekanism-Feature-Requests/issues/343#issuecomment-1013941292
